app/models/like.rb

```diff
@@ -1,7 +1,6 @@
 class Like < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
-
```

Comment:
- [ ]  It would be nice if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#like_count` method every time a new like has been saved to database, then make the `like_count` method private.

```ruby
 after_save :like_count

  private

  def like_count
    post.increment!(:likes_count)
  end
```